### PR TITLE
Updated minikube deployment document

### LINKED ISSUE #19: Create the Minikube deployment and getting started document.
Move over the instructions from the Kubeflow repo and organize the instructions inline as much of the setup as possible. Would be good to actually run through them to ensure they are still valid and up-to-date.

### PR DESCRIPTION
**Pull request by abhi-g**
Added installation steps for Minikube, and for deploying Kubeflow using bootstratpper. Resolves #19

### CODE REVIEW

File: content/docs/started/getting-started-minikube.md [hunk 1]
```diff
@@ -2,12 +2,174 @@
...
+```
+
+Setup port forwarding for the central dashboard UI and Jupyter Hub
+```
+$ POD=`kubectl -n kubeflow get pods --selector=service=ambassador | awk '{print $1}' | tail -1`
+$ kubectl -n kubeflow port-forward $POD 8080:80 2>&1 >/dev/null &
+$ POD=`kubectl -n kubeflow get pods --selector=app=tf-hub | awk '{print $1}' | tail -1`
+$ kubectl -n kubeflow port-forward $POD 8000:8000 2>&1 >/dev/null &
+```
+Now you can access the Kubeflow ashboard at http://localhost:8080/ and JupyterHub at http://localhost:8000/.
```
Comment (pdmack) [inline content/docs/started/getting-started-minikube.md]: "dashboard"

Comment (abhi-g) [reply content/docs/started/getting-started-minikube.md]: good catch! thanks

**Pull request merged**